src/pcre2_pattern_info.c

```diff
@@ -289,8 +289,7 @@ if (re->magic_number != MAGIC_NUMBER) return PCRE2_ERROR_BADMAGIC;
 if ((re->flags & (PCRE2_CODE_UNIT_WIDTH/8)) == 0) return PCRE2_ERROR_BADMODE;
 
 cb.version = 0;
-cc = (PCRE2_SPTR)((const uint8_t *)re + sizeof(pcre2_real_code))
-     + re->name_count * re->name_entry_size;
+cc = (PCRE2_SPTR)((uint8_t *)re + re->code_start);
```

Comment:
Zoltan, I tagged you on this review just for this one line, really.

I'm sure it's correct, but I guess I wanted to give you a chance to say whether you're happy or not with releasing the fix in this way, waiting until the next release (basically: it's a nasty crash, but we don't need to do a security advisory, because no-one uses this function).

Reply:
I am happy with it. We don't need another security release.